docs/application/dotnet/guides/nlp/nlp.md

```diff
@@ -0,0 +1,284 @@
+3.  When nlp object is no longer needed, call Dispose() to release resource of Nlp:
+
+    ```
+    protected override void OnTerminate()
+        {
+         ..
+         nlp.Dispose();
+         ..
+        }
+    ```
+
+
+<a name="ne_chunk_result"></a>
+## To get the tokens and tags of sentence 
+
+To recieve the tokens and tags from sentence:
+
+1.  Construct an NaturalLanguageProcess on OnCreate(), and Connect the Nlp service on init of app :
+
+    ```
+    protected override void OnCreate
+    {
+        ..
+        nlp = new NaturalLanguageProcess();
+        Task task = nlp.Connect();
+        ..
+    }
+    ```
+
+2.  Call the NamedEntityRecognitionAsync(string msg) in a async Task method:
+
+    ```
+    public async Task OnTokenButtonPressedAsync(string msg)
+    {
+        ..
+        var ret = nlp.NamedEntityRecognitionAsync(msg);
+        NamedEntityRecognitionResult nr = await ret;
+        ..
+    }
+    ```
+
+3.  When it is no longer needed, call Dispose() to release resource of Nlp:
+
+    ```
+    protected override void OnTerminate()
+        {
+         ..
+         nlp.Dispose();
+         ..
+        }
+    ```
+
+
+
+
+<a name="lemmatize_result"></a>
+## To get the actual word of sentence 
+
+To recieve the actual word from sentence:
+
+1.  Construct an NaturalLanguageProcess object on OnCreate(), and Connect the Nlp service on init of app :
+
+    ```
+    protected override void OnCreate
+    {
+        ..
+        nlp = new NaturalLanguageProcess();
+        Task task = nlp.Connect();
+        ..
+    }
+    ```
+
+2.  Call the LemmatizeaAsync(string msg) in a async Task method:
+
+    ```
+    public async Task OnLemmButtonPressedAsync(string msg)
+    {
+        ..
+        var ret = nlp.LemmatizeaAsync(msg);
+        LemmatizeResult lr = await ret;
+        ..
+    }
+    ```
+
+3.  When it is no longer needed, call Dispose() to release resource of Nlp:
+
+    ```
+    protected override void OnTerminate()
+        {
+         ..
+         nlp.Dispose();
+         ..
+        }
+    ```
+
+
+
+<a name="langdetect_result"></a>
+## To get the language of sentence 
+
+To recieve the language from sentence:
+
+1.  Construct an NaturalLanguageProcess on OnCreate(), and Connect the Nlp service on init of app :
+
+    ```
+    protected override void OnCreate
+    {
+        ..
+        nlp = new NaturalLanguageProcess();
+        Task task = nlp.Connect();
+        ..
+    }
+    ```
+
+2.  Call the LanguageDetectAsync(string msg) in a async Task method:
+
+    ```
+    public async Task OnLangButtonPressedAsync(string msg)
+    {
+        ..
+        var ret = nlp.LanguageDetectAsync(msg);
+        LanguageDetectedResult lr = await ret;
+        ..
+    }
+    ```
+
+3.  When it is no longer needed, call Dispose() to release resource of Nlp:
+
+    ```
+    protected override void OnTerminate()
+        {
+         ..
+         a.Dispose();
+         ..
+        }
+    ```
+
+
+
+## Related Information
+* Dependencies
```

Comment:
May I know whether this dependency is for mobile or wearable?
To maintain consistency, as per other docs, it has to be specified as below:
Dependencies:
- Tizen  4.0 and Higher for Mobile

- Tizen  4.0 and Higher for Wearable

Reply:
We don't divide profile information for **dotnet** guides.
If you check other guide pages under dotnet, you'd see there are no profile information also.

So, this dependencies should keep current status.